server/controllers/index.js

```diff
@@ -27,7 +27,26 @@ function parseGet(request, response) {
 }
 
 function parsePost(request, response) { } // eslint-disable-line
-function parsePut(request, response) { } // eslint-disable-line
+
+function parsePut(request, response) {
+  const newURL = process.env.API_URL + request.url.slice(4);
+  const config = {
+    headers: {
+      Authorization: process.env.API_TOKEN,
+    },
+  };
+
+  axios.put(newURL, request.body, config)
+    .then((apiRes) => {
+      console.info('request ended in', apiRes.status);
+      response.status(apiRes.status);
+      response.send(apiRes.data);
```

Comment:
Could we adjust this to the below (per Learn): 

Status: 204 NO CONTENT

Reply:
... as above.

Reply:
See response above.